src/common/404/PageNotFound.jsx

```diff
@@ -1,25 +1,46 @@
+import { useState, useEffect } from "react";
 import { ReactComponent as Image404 } from "images/img-404.svg";
 import "./404.css";
 import Loader from "common/spinner/spinner";
+import { useNavigate } from "react-router-dom";
 
 const PageNotFound = ({ loading, msg, details, Image }) => {
+  const [timer, setTimer] = useState(5);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimer(timer - 1);
+    }, 1000);
```

Comment:
Can we add comments here like what we trying to do?

Reply:
it is to show the timer to redirect back if user reaches not found page somehow.